bom/application/pom.xml

```diff
@@ -5051,9 +5051,14 @@
             </dependency>
             <dependency>
                 <groupId>org.hibernate</groupId>
-                <artifactId>quarkus-local-cache-jakarta</artifactId>
+                <artifactId>quarkus-local-cache</artifactId>
                 <version>${hibernate-quarkus-local-cache.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.hibernate.common</groupId>
+                <artifactId>hibernate-commons-annotations</artifactId>
+                <version>${hibernate-commons-annotations.version}</version>
+            </dependency>
```

Comment:
Hey @Sanne, it looks like this dependency got added by accident - it is already in the pom just three deps above 🙈 
I was trying to build your branch, and maven "noticed" that 😃.

Reply:
btw, just in case I'm looking at the tests in `quarkus-integration-test-main`

Reply:
ah thanks! will fixup

Reply:
pushed